evictiontest/workflow_cache_eviction_test.go

```diff
@@ -176,6 +177,7 @@ func (s *CacheEvictionSuite) TestResetStickyOnEviction() {
 	workflowWorker := internal.NewWorker(s.service, "test-domain", "tasklist", worker.Options{
 		DisableActivityWorker: true,
 		Logger:                zaptest.NewLogger(s.T()),
+		IsolationGroup:        "zone-1",
```

Comment:
is this necessary for this test, or just adding a value to have it in a test somewhere?

Reply:
the wrapper code-path is checking if this field is populated before using the additional wrapper:
```
	if options.IsolationGroup != "" {
		service = isolationgroup.NewWorkflowServiceWrapper(service, options.IsolationGroup)
	}

```
. So I figured to give a more realistic test in a prod-ish env we can excecise this additional codepath